src/OSPSuite.SimModelNative/include/SimModel/PInvokeQuantity.h

```diff
@@ -16,6 +16,11 @@ namespace SimModelNative
       
       //<array> has preallocated memory for <size> elements
       SIM_EXPORT void FillQuantityValues(Quantity* quantity, double* values, int size, bool& success, char** errorMessage);
+
+      //fills senistivity values of variable <quantity> for parameter given by <parameterPath>
+      //<values> arrays is pre-allocated with <size> elements
+      SIM_EXPORT void FillSensitivityValues(Quantity* quantity, double* values, int size, const char* parameterPath, bool& success, char** errorMessage);
+
```

Comment:
`FillSensitivityValues `was defined in the wrong header and the corresponding symbol definition was not exported by the DLL.